generators/dotnet-templates/Microsoft.BotFramework.CSharp.CoreBot/content/CoreBot.Tests/CoreBot.Tests.csproj

```diff
@@ -21,8 +21,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.App" />
-    <PackageReference Include="Microsoft.Bot.Builder.Testing" Version="4.7.0" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="3.1.1" />
+    <PackageReference Include="Microsoft.Bot.Builder.Testing" Version="4.8.0-preview-200219-106986" />
```

Comment:
This needs to be the final shipping assembly, not the preview assembly

Reply:
Yes - I have them on the preview assemblies right now until the final assemblies are published.